src/panels/logbook/ha-panel-logbook.ts

```diff
@@ -184,6 +191,15 @@ export class HaPanelLogbook extends LitElement {
     }
   }
 
+  private async _fetchUsers() {
```

Comment:
We need to compute this in `_getData` and only set loading=false once users are resolved. We can do this by storing the promise that is returned by `this._fetchUsers` and using `await Promise.all([])`. It's late here now but I can fix that tomorrow 👍

Reply:
Any assistance here would be appreciated as I'm still getting my feet wet on the frontend.  Thanks @balloob